test/unit/app/browser/api/ledgerTest.js

```diff
@@ -586,40 +590,36 @@ describe('ledger api unit tests', function () {
         assert(saveVisitSpy.notCalled)
       })
 
-      it('min duration is set to minimum visit time if below that threshold', function () {
-        const xhr2 = 'https://www.youtube.com/api/stats/watchtime?docid=kLiLOkzLetE&st=20.338&et=21.339'
-        ledgerApi.onMediaRequest(cacheAppState, xhr2, ledgerMediaProviders.YOUTUBE, 1)
-        assert(publisherFromMediaPropsSpy.notCalled)
-        assert(saveVisitSpy.withArgs(cacheAppState, publisherKey, paymentsMinVisitTime, false).calledOnce)
-      })
-
-      it('min duration is set to minimum visit time if below that threshold (string setting)', function () {
-        paymentsMinVisitTime = '5000'
```

Comment:
Was this intentionally removed? I think it was a good test to have (making sure string setting is converted)

Reply:
yeah, because now we don't have this threshold anymore. We are sending actual milliseconds in that we receive from youtube.

Reply:
We are sending `ignoreMinTime: true` into `addPublisher`, so we don't need to set min visit time anymore

Reply:
👍